<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="3">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="4">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>27</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: black;
	border-radius: 10px;
	color: #ffffff;
}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="ButtonMul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>Ã—</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="ButtonPlus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Delete">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>DEL</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="ButtonMinus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="ButtonAC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>26</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="ButtonEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="ButtonDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>Ã·</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="squareRoot">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>45</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>âˆš</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="copyright">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>45</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-radius: 40px;
	padding: 5px;
	color: white;
}
QPushButton:pressed {
	background-color: #A9A9A9;
}</string>
      </property>
      <property name="text">
       <string>Â©</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
